libraries/dotnet/EnergyOrigin.TokenValidation/EnergyOrigin.TokenValidation/B2C/TermsRequirementHandler.cs

```diff
@@ -0,0 +1,31 @@
+using EnergyOrigin.TokenValidation.b2c;
+using Microsoft.AspNetCore.Authorization;
+
+namespace EnergyOrigin.TokenValidation.B2C;
+
+[AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)]
+public class DisableTermsRequirementAttribute : Attribute;
```

Comment:
Why do we need a DisableTermsRequirementAttribute? If not terms policy is choosen, then it shouldn't be checked?

Reply:
Default behaviour should be to lock down all apis' from users unless terms are accepted. This way we don't have to write add a new terms policy in all controllers. we just disable the ones we want. Opt-out rather than opt-in.